src/main/kotlin/com/safehill/kclient/api/dtos/SHRecipientEncryptionDetailsDTO.kt

```diff
@@ -1,8 +1,13 @@
 package com.safehill.kclient.api.dtos
 
+import kotlinx.serialization.Serializable
+
+@Serializable
 data class SHRecipientEncryptionDetailsDTO(
-    val userIdentifier: String,
+    val recipientUserIdentifier: String,
     val ephemeralPublicKey: String, // base64EncodedData with the ephemeral public part of the key used for the encryption
     val encryptedSecret: String, // base64EncodedData with the secret to decrypt the encrypted content in this group for this user
-    val secretPublicSignature: String // base64EncodedData with the public signature of the user sending it
+    val secretPublicSignature: String, // base64EncodedData with the public signature of the user sending it
+    val senderPublicSignature: String
```

Comment:
Add documentation for it (you'll find it in the Swift counterpart)

Reply:
Sure